### ISSUE TITLE
Differentiating between draging over the first drop slot and dragging over a node

### ISSUE THREAD

**Issue by Gragx** [500tech/angular-tree-component#632]
Hello there

I need to differentiate between dragging a node over the first drop slot in a parent and dragging it directly over a parent. The usecase is, that I want dropped nodes as the last child within their parent. 

The drop logic is already doable using the dropOnNode flag in the drop event. 
What I need is additional info already in allowDrop, so I can prevent dragging a last child onto its parent.

So basically I'd like to have a dropOnNode parameter in allowDrop

**Comment by adamkleingit** [comment]
Hi,
I recently opened a slack channel for the library's users community.
I think it will be a good place for the community to discuss different ways of implementation, to ask for feature requests, collaborate on PRs, and hear about bugfixes and releases.

I invite you to join:
https://angular-tree-component.herokuapp.com/
